src/openforms/contrib/zgw/service.py

```diff
@@ -52,22 +55,22 @@ def create_zaak(
 
 def default_get_drc() -> Service:
     config = ZgwConfig.get_solo()
-    return config.drc_service
+    zgw_config = config.default_zgw_api_group
+    return zgw_config.drc_service
 
 
-def partial_update_zaak(zaak_url: str, data: dict) -> dict:
-    config = ZgwConfig.get_solo()
-    client = config.zrc_service.build_client()
+def partial_update_zaak(zaak_url: str, data: dict, zrc_service: Service) -> dict:
```

Comment:
The order of arguments is a bit inconsistent - `create_zaak` takes the service as first argument, this function as last. Can you normalize that?

I'm also wondering if it wouldn't make more sense to pass the `client` instead of the service? This would unlock some connection pooling with constructs like:

```py
with zrc_service.build_client() as client:
    partial_update_zaak(client, ...)
    ... # other operations with same client, (eventually) sharing a connection pool
```